ci-operator/step-registry/sandboxed-containers-operator/install-trustee-operator/sandboxed-containers-operator-install-trustee-operator-commands.sh

```
@@ -135,54 +123,53 @@ function wait_until() {
     return 1
   fi
 
-  git clone --depth 1 --branch "${TRUSTEE_CHARTS_REF}" "${TRUSTEE_CHARTS_REPO}" "${charts_dir}"
+  echo ">>> Extracting charts and helm from image: ${HELM_CHART_IMAGE}" >&2
+
+  mkdir -p "${charts_dir}" "${bin_dir}"
+  local extract_output
+  local lib_dir="${SCRATCH}/lib"
+  mkdir -p "${lib_dir}"
+  if extract_output=$(oc image extract "${HELM_CHART_IMAGE}" \
+    --path /charts/:${charts_dir}/ \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)